### DISCUSSION TITLE
邮箱填错了可以改吗？

### DISCUSSION THREAD

**Discussion by posdem** [getlantern/lantern-forum-cn#879] [Q&A] [answered]
注册时没有注意，等到手机3登录时才发现 注册邮箱少写了一数，现在手机用不了，不知道可不可以修改绑定的邮箱

**Comment by MaxMKMax** [comment]
@posdem 可以改的. 请您提供我支付单据和错的账号邮箱. 和您想改的邮箱. 我帮您提交上去.

**Reply by posdem** [reply, marked as answer]
错误邮箱:115801520@qq.com
正确邮箱:1158301520@qq.com
付款账号：622812****3028 邮政信用卡
支付时间 9-19 10.46 金额340
![Screenshot_2024-09-22-12-11-56-58_dec8cbdd3d4bbcb4afd5902dbbc3a11c](https://github.com/user-attachments/assets/4aa9c733-7717-4efd-adc8-62edfcd73d4b)
